src/utils/fetch-lybsin-podcasts.js

```diff
@@ -0,0 +1,53 @@
+import parser from 'fast-xml-parser';
+import dlv from 'dlv';
+
+// Fetches and parses podcast info from https://mongodb.libsyn.com/rss
+
+const RSS_URL = `https://mongodb.libsyn.com/rss`;
+
+export const simplifyPodcast = podcast => {
```

Comment:
Are we `export`ing these two other functions for any reason?

Reply:
I was only exporting them because I was following the `fetch-event-data.js` file. Just removed them since we're won't  be calling them from anywhere else.